packages/next/build/index.ts

```diff
@@ -100,15 +100,17 @@ export default async function build(dir: string, conf = null): Promise<void> {
       })
     : 'noop'
 
+  if (!config.keepPastBuilds || isFlyingShuttle) {
+    await recursiveDelete(distDir, /^(?!cache(?:[\/\\]|$)).*$/)
+    await recursiveDelete(path.join(distDir, 'cache', 'next-minifier'))
```

Comment:
`.next/cache` should be preserved in my opinion, otherwise subsequent builds will get slower.

Reply:
Removing cache was specific to flying shuttle because it'd generate megabytes large JS files. With granular chunking or even current behavior, it's really not a problem.